go.mod

```diff
@@ -2,26 +2,23 @@ module github.com/apache/skywalking-cli
 
 go 1.13
 
+replace github.com/apache/skywalking-swck => github.com/hanahmily/skywalking-swck v0.0.0-20210106071233-5406527abfa3
```

Comment:
What is this?

Reply:
To dependent my private dev branch temporarily which doesn't be merged, that's why I mark this PR as 'WIP'